
project external/noto-fonts/
diff --git a/external/noto-fonts/Android.mk b/external/noto-fonts/Android.mk
index f6e19ff..1fb1632 100644
--- a/external/noto-fonts/Android.mk
+++ b/external/noto-fonts/Android.mk
@@ -86,7 +86,8 @@ font_src_files := \
     NotoSerif-Regular.ttf \
     NotoSerif-Bold.ttf \
     NotoSerif-Italic.ttf \
-    NotoSerif-BoldItalic.ttf
+    NotoSerif-BoldItalic.ttf \
+    NotoColorEmoji.ttf
 
 #############################################################################
 # The following fonts are excluded from SMALLER_FONT_FOOTPRINT builds.
